types/react-devtools-inline/commons.d.ts

```diff
@@ -43,13 +43,13 @@ export type FetchFileWithCaching = (url: string) => Promise<string>;
 export type Context = FetchFileWithCaching | null;
 
 export interface DevtoolsProps {
-    bridge: FrontendBridge;
+    bridge?: FrontendBridge;
```

Comment:
not optional
![image](https://user-images.githubusercontent.com/56086371/196882743-a72c066e-b9ef-4681-b976-adea97be5ea4.png)